### PR TITLE
Jufa/address sudo feedback

### PR DESCRIPTION
**Pull request by jfaltenbacher**
- Removing admin content
-  troubleshoot removed
- fix lonely sections
- A bit of restructuring and rewriting
- adding examples

### CODE REVIEW

**Comment by jfaltenbacher** [comment]
@janajaeger  : please add link to your article in note of section _1.2 sudo on SUSE Linux sytems_  before merging

**Review by taroth21 — CHANGES_REQUESTED** [review]
@jfaltenbacher : Many thanks, this is much more straightforward than the previous 'combined' article for multiple target groups! 

Nevertheless I have left some suggestions in a commented PDF on how to move some content around (please also make sure to hand over to Daria for editorial review next).

**Comment by taroth21** [comment]
@jfaltenbacher : Please find my feedback here:
[sudo-run-commands-as-superuser_feedback_taroth.pdf](https://github.com/SUSE/doc-modular/files/12819158/sudo-run-commands-as-superuser_feedback_taroth.pdf)

**Comment by jfaltenbacher** [comment]
@dariavladykina : please perform a language review. Final article as PDF attached:
[sudo-run-commands-as-superuser_en.pdf](https://github.com/SUSE/doc-modular/files/12843828/sudo-run-commands-as-superuser_en.pdf)

File: tasks/sudo-run-command.xml [hunk 1]
```diff
@@ -22,60 +22,73 @@
...
+      file into a directory where only the &rootuser; user has editing rights.
+    </para>
     <step>
       <para>
-        To show the content of the <filename>sudoers</filename> file, enter the
-        following command:
+        Open a terminal and create a test file with:
       </para>
-<screen>&prompt.user;&sudo; cat /etc/sudoers</screen>
+      <screen>&prompt.user;<command>touch text.txt</command></screen>
```
Comment (dariavladykina) [inline tasks/sudo-run-command.xml:52]:
```suggestion
      <screen>&prompt.user;<command>touch test.txt</command></screen>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please note some suggestions here. Thanks!

File: concepts/sudo-concept.xml [hunk 2]
```diff
@@ -8,51 +11,146 @@
...
-
-      As a normal user on Linux, you have reduced permissions. For example, you cannot write to system directories.
-      For security reasons, the normal user is separate from the &rootuser; user.
-      You must have &rootuser; privileges to run commands which can only be executed
-      by the &rootuser;.
-      The following options to log in as &rootuser; are available:
+    &sudo; is a Linux command that is used to temporarily execute programs as
+    another user.
+    It is the abbreviation for <literal>substitute user and do</literal> and
+    borrows the privileges of another user, for example the &rootuser; user.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:39]: fixed as suggested

File: concepts/sudo-concept.xml [hunk 3]
```diff
@@ -8,51 +11,146 @@
...
+    This way, &sudo; helps you accomplish system administration tasks without
+    logging in as &rootuser;.
+  </para>
+  <section xml:id="sudo-basic-concepts-general">
+    <title>&sudo; on Linux systems in general</title>
+  <para>
+    As a regular user on Linux, you have reduced permissions which is sufficient
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:46]: fixed as suggested

File: concepts/sudo-concept.xml [hunk 4]
```diff
@@ -8,51 +11,146 @@
...
+    for most of the tasks. The &rootuser; user is the Linux superuser and the
+    equivalent to the administrator.
+  </para>
+  <para>
+    &sudo; allows specified (non-&rootuser;) users or a group of users, to run
+    privileged tasks, like calling system functions as &rootuser; or another
+    user. Through a configuration file, the system administrator can control,
+    which users are allowed to run &sudo; commands and for which tasks.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:54]: fixed as suggested

File: concepts/sudo-concept.xml [hunk 5]
```diff
@@ -8,51 +11,146 @@
...
+    from the &rootuser; user.
+    As a regular user, you cannot write to system directories or install programs.
+    Any user with access to the &rootuser; password can gain this privileges and
+    accidentally or maliciously break the system.
+  </para>
+  <para>
+    For example, you are logged in as regular user and accidentally browsing to
+    a malicious website. An attacker cannot install malware on your system
+    behind your back, as you are logged in as regular user, means without any
+    administrative permissions.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:70]: fixed as suggested

File: concepts/sudo-concept.xml [hunk 6]
```diff
@@ -8,51 +11,146 @@
...
+      session. So you do not need to provide the &rootuser; password again when
+      running another &sudo; command.
+    </para>
+    <para>
+    To run a command with &sudo;, this syntax is used:
+  </para>
+  <screen>
+    &prompt.user;<command>sudo [command]</command>
+      </screen>
+  <para>The following example explains, how the &sudo; command is used, to open
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:117]: fixed as suggested

File: concepts/sudo-concept.xml [hunk 7]
```diff
@@ -8,51 +11,146 @@
...
+&prompt.user;<command>yast</command>
+Absolute path to 'yast' is '/usr/sbin/yast', so running it may require superuser privileges (eg. root).<co xml:id="co-sudo-usage-yast-answer"/>
+&prompt.sudo;<command>yast</command>
+[sudo] password for root:<co xml:id="co-sudo-usage-pw"/>
+         </screen>
+         <calloutlist>
+          <callout arearefs="co-sudo-usage-yast-answer">
+           <para>
+            When only the <command>yast</command> command is entered, the system
+            returns, that &rootuser; privileges are required.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:134]: fixed as suggested

File: tasks/sudo-run-command.xml [hunk 8]
```diff
@@ -22,60 +22,73 @@
...
-    following example shows how to execute a command prefaced with &sudo;.
+    Depending on the &sudo; configuration, the elevated privileges persist for a
+    certain period of time and as long as you are working in the same terminal
+    session. So you do not need to provide the &rootuser; password again when
+    running another &sudo; command.
   </para>
   <procedure>
     <title>Running a command prefaced with &sudo;</title>
+    <para>
+      The following procedure describes how to use the &sudo; command, to copy a
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply tasks/sudo-run-command.xml:45]: fixed as suggested

File: tasks/sudo-run-command.xml [hunk 9]
```diff
@@ -22,60 +22,73 @@
...
-        is displayed.
+       Now, enter the same command prefaced with &sudo;:
       </para>
+<screen>&prompt.sudo;<command>cp test.txt /usr/local/bin</command></screen>
+    </step>
+    <step>
       <para>
-        If you do not have the required &sudo; privileges or you run the
-        command not prefaced with &sudo;, the following message returns:
+        You are prompted to enter the &rootuser; password. Note, that the
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply tasks/sudo-run-command.xml:77]: fixed as suggested

File: concepts/difference-between-sudo-and-su.xml [hunk 10]
```diff
@@ -7,31 +7,90 @@
...
+  <listitem>
+    <para>
+     <command>su</command> allows you to run a command as &rootuser; but
+     requires you to know the &rootuser; password. 
+   </para>
+   </listitem>
+   <listitem>
+    <para>
+      It cannot be configured to restrict access to certain users and tasks and
+      elevates the privileges for anyone who knows the &rootuser; password.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/difference-between-sudo-and-su.xml:79]: fixed as suggested

File: tasks/sudo-run-command.xml (hunk 1 above)
Comment (jfaltenbacher) [reply tasks/sudo-run-command.xml:52]: fixed as suggested, good catch

File: concepts/sudo-concept.xml [hunk 11]
```diff
@@ -8,51 +11,146 @@
...
+    which users are allowed to run &sudo; commands and for which tasks.
+  </para>
+</section>
+<section xml:id="sudo-basic-concepts-suse">
+  <title>&sudo; on &productname; sytems</title>
+  <para>
+    For security reasons, all &productname; systems separate the regular user
+    from the &rootuser; user.
+    As a regular user, you cannot write to system directories or install programs.
+    Any user with access to the &rootuser; password can gain this privileges and
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:63]: fixed as suggested

File: concepts/sudo-concept.xml [hunk 12]
```diff
@@ -8,51 +11,146 @@
...
+         <calloutlist>
+          <callout arearefs="co-sudo-usage-yast-answer">
+           <para>
+            When only the <command>yast</command> command is entered, the system
+            returns, that &rootuser; privileges are required.
+           </para>
+          </callout>
+          <callout arearefs="co-sudo-usage-pw">
+           <para>
+            Starting the <command>yast</command> command with &sudo;, prompts
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply concepts/sudo-concept.xml:139]: fixed as suggested

File: tasks/sudo-run-command.xml [hunk 13]
```diff
@@ -22,60 +22,73 @@
...
-        password is not shown during input, either as clear text or as
-        masking characters.
+        Try to copy the file into the <filename>usr/local/bin/</filename>
+        directory with:
       </para>
-<screen>password for root:</screen>
-    </step>
+      <screen>&prompt.user;<command>cp test.txt /usr/local/bin</command></screen>
+      <para>
+        The terminal returns for example:
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply tasks/sudo-run-command.xml:61]: fixed as suggested

**Pull request merged**